public/index.html

```diff
@@ -85,7 +85,7 @@
 
         </div>
         <div class="subhead">
-          <div class="subhead-left i18n" data-i18n="ftm-index-need-helpers">Let's Help The Helpers</div>
+          <div class="subhead-left i18n mr-lg-3 py-4 px-3" data-i18n="ftm-index-need-helpers">Let's Help The Helpers</div>
```

Comment:
Why do you need the additional classes? Can't you just add the relevant margins+padding to the .subhead-left rule?

Reply:
The .py-4 padding top/bottom is pretty hefty on mobile.

![image](https://user-images.githubusercontent.com/743030/78574596-0ac02b80-77df-11ea-93db-4355447669bc.png)

Reply:
We don't really. What are your thoughts here? I'm thinking we may as well just try and lean into bootstrap styling where possible since it's already defined which is why I switched to their padding styles. I'll make a tweak to the padding on mobile and merge but can follow up tonight